opengrok-web/src/main/webapp/js/utils-0.0.29.js

```diff
@@ -2390,3 +2390,23 @@ function getCookie(cname) {
     }
     return "";
 }
+
+//polyfills for IE10-11
+// References:
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger
+
+
+if (!Number.isInteger)
+{
+    Number.isInteger = function numberIsInteger(value)
+    {
+        return typeof value === 'number' && isFinite(value) && Math.floor(value) === value;
```

Comment:
there is only one match, isn't it better to do it inline or use some kind of alternative with parseInt() === NaN ?

Reply:
if caretPos need only simple number check,
i can change to 
```
typeof caretPos === 'number'
```